packages/astro/test/server-islands.test.js

```diff
@@ -37,6 +37,11 @@ describe('Server islands', () => {
 				assert.equal(serverIslandEl.length, 0);
 			});
 
+			it('island is not indexed', async () => {
+				const res = await fixture.fetch('/_server-islands/Island');
```

Comment:
this is failing because you're not providing a body. Change it to a POST and do like in the other test, here: https://github.com/withastro/astro/blob/2d02170611a1d75e7dad00e1b86ffbb962c0b699/packages/astro/test/server-islands.test.js#L48-L52

Reply:
I fixed tests and `X-Robots-Tag: noindex` is present even without my fix

Reply:
I don't see how, there is not `X-Robots-Tag` in the codebase.